src/main/resources/public/js/activity.service.js

```diff
@@ -54,45 +54,46 @@
           addTimeForKm(activity, seconds, distanceHalfMarathon);
         }
         // calculate time for every distance starting with nearest km (round ceil)
-        for (var i = Math.ceil(activity.distance); i >= 1; i--) {
+        for (let index = Math.ceil(activity.distance); index >= 1; index -= 1) {
```

Comment:
Eslint, see http://eslint.org/docs/rules/no-plusplus and https://github.com/airbnb/javascript#variables--unary-increment-decrement.